lib/widget/reusable_zoomable_widget.dart

```diff
@@ -5,36 +5,27 @@ class ZoomablePhoto extends StatelessWidget {
   // Image url for the graph
   final String imageUrl;
   final Color backgroundColor;
-  final double padding;
 
-  const ZoomablePhoto({this.imageUrl, this.backgroundColor, this.padding = 20});
+  const ZoomablePhoto({this.imageUrl, this.backgroundColor});
 
   @override
   Widget build(BuildContext context) {
-    return Padding (
-      padding: const EdgeInsets.only(top: 20.0),
-      child: Column(
-          crossAxisAlignment: CrossAxisAlignment.stretch,
-          children: <Widget>[
-            Expanded(
-              child: ClipRect(
-                child: PhotoView(
-                  backgroundDecoration: BoxDecoration(
-                    color: backgroundColor
-                  ),
-                  imageProvider: AssetImage(imageUrl),
-                  customSize: _getInitialZoomableImageSize(context, padding),
-                  basePosition: Alignment.topCenter,
-                ),
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.stretch,
+      children: <Widget>[
+        Expanded(
+          child: ClipRect(
+            child: PhotoView(
+              backgroundDecoration: BoxDecoration(
+                color: backgroundColor
               ),
-            )
-          ]
-        ),
+              imageProvider: AssetImage(imageUrl),
+              initialScale: PhotoViewComputedScale.contained * 0.9,
+              basePosition: Alignment.topCenter * 0.9,
```

Comment:
I wanted to tag @workerbee22 on this to get suggestions.  What this is trying to achieve is add some padding at the top of the image, as is shown in the designs:

![image](https://user-images.githubusercontent.com/902252/78417897-af792680-7682-11ea-9149-486aa8d0540f.png)

When I run the code though, because the `0.9` thing is a percentage, it seems to not quite work out for different sized images.

@workerbee22 is there a way to just add 16dp of padding here?

Reply:
@workerbee22 The problem is this third party library `photo_view` does percentage setting rather than points. And the paddings around the image needs to disappear when zoom in.

Reply:
I'm happy to leave this for a later issue as its not a major thing.  If @workerbee22 agrees we can get your PR merged and fix this up a low priority design tweak

Reply:
Thats OK. Also not sure why we wrap it with `ClipRect` ? That is an expensive operation.

Reply:
We'll probably have to look at this separatly.